src/specs/api.v2.yaml

```diff
@@ -76,21 +107,106 @@ paths:
           content:
             application/json:
               schema:
-                $ref: ./models/HTTPSuccess.v1.yaml
+                $ref: '#/components/schemas/HTTPSuccess'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPError'
+        '401':
+          description: The request lacks authentication credentials.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPError'
+    patch:
+      summary: Patch an experiment
+      description: Update some properties in an experiment
+      operationId: patchExperiment
+      x-eov-operation-id: experiment#patchExperiment
+      x-eov-operation-handler: routes/experiment
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ExperimentPatch'
+      responses:
+        '200':
+          description: Create experiment
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPSuccess'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPError'
+        '401':
+          description: The request lacks authentication credentials.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPError'
+        '404':
+          description: Not found error.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPError'
+  '/experiments/{experimentId}/samples/position':
+    put:
+      summary: Update position for a sample
+      description: Set a new position for a sample
+      operationId: updateSamplePosition
+      x-eov-operation-id: experiment#updateSamplePosition
+      x-eov-operation-handler: routes/experiment
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                oldPosition: 
+                  type: integer
+                newPosition:
+                  type: integer
```

Comment:
you could explicitly add here "additionalProperties: false" to not allow passing of any parameters outside of the schema